src/main/java/racingcar/utils/message/GuideMessage.java

```diff
@@ -0,0 +1,8 @@
+package racingcar.utils.message;
+
+public class GuideMessage {
```

Comment:
생성자 막고 final class로 상속 못하게 하기
```suggestion
private GuideMessage {}
```